client/src/containers/legend-settings/index.tsx

```diff
@@ -0,0 +1,100 @@
+'use-client';
+
+import { useCallback } from 'react';
+
+import { Eye, EyeOff, PaintBucket, XCircle } from 'lucide-react';
+
+import { LayerSettings } from '@/types/layers';
+
+import { useSyncLayers } from '@/hooks/datasets/sync-query';
+
+import { Popover, PopoverContent, PopoverTrigger } from '@/components/ui/popover';
+import { Slider } from '@/components/ui/slider';
+import { Tooltip, TooltipContent, TooltipTrigger } from '@/components/ui/tooltip';
+
+const LegendSettings = ({ settings }: { settings: LayerSettings }) => {
```

Comment:
When select two layers i see no legends in map. 
Also Land degradation legend is without styles.
![Screenshot 2024-03-21 at 11 24 00](https://github.com/Vizzuality/afoco/assets/51995866/154c07fc-9085-4583-8dd3-bc3259076d2b)